synapse/cores/common.py

```diff
@@ -453,16 +456,18 @@ def _addDataModels(self, modtups):
         for name, modl in modtups:
 
             for tnam, tnfo in modl.get('types', ()):
-                self.addRuntNode('syn:type', tnam, **tnfo)
+                tdef = self.getTypeDef(tnam)
+                self.addRuntNode('syn:type', tnam, tdef[1])
```

Comment:
since props is a kwarg, lets use the name here in case of any ordering shinanigans

Reply:
fixed